XenModel/Actions/Updates/XenServerVersion.cs

```diff
@@ -85,6 +86,7 @@ public class XenServerVersion
             BuildNumber = buildNumber;
             PatchUuid = patchUuid;
             PresentAsUpdate = presentAsUpdate;
+            MinimumXcVersion = minXcVersion;

```

Comment:
I think that MinimumXcVersion  should be of type Version, so that when we compare it with the items in XenCenterVersions we don't do a string comparision (as strings "7.4.0" and "7.4" are different)